app/assets/stylesheets/common/base/user-badges.scss

```diff
@@ -143,6 +143,12 @@
     font-size: $font-up-2;
   }
 
+  .favorite {
```

Comment:
what does it look like on mobile please?

Reply:
That's a fair question and I should definitely have included screenshots! Sorry about, I think I spent too much time in my last role working on a desktop-only product 😅 The screenshot below is a fullpage screenshot on a Pixel 2 width.

![image](https://user-images.githubusercontent.com/3116474/110772769-461f7180-825c-11eb-9a81-c4ace2979739.png)

Reply:
We should probably add some styling/color/emoji to differentiate "favorite" and "unfavorite"

Reply:
Sure! How about https://fontawesome.com/icons/star?style=solid and https://fontawesome.com/icons/star?style=regular depending on state?

Reply:
Sure try with that. Also can you make sure the button doesn't overlap? Maybe if it's in the top-right hand corner it would be easier?

Reply:
There's an issue with placing it in the top right corner, which @SamSaffron pointed out in the discourse thread: it could obscure the badge count. Although we could swap their positioning, placing the badge count on the top left and the favorite/unfavorite action on the top right. What do you think?